src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (salary) {
+    let cleanedSalary = salary.slice(1);
+
+    cleanedSalary = cleanedSalary.split(',').join('');
+
+    return Number(cleanedSalary);
+  }
+
+  return 0;
+}
+
+function sortList(list) {
+  const listItem = [...list];
+
+  return listItem.sort((a, b) => getSalary(b) - getSalary(a));
+}
+
+listEmployees.innerHTML = '';
+
+const sortedList = sortList(employees);
+
+sortedList.forEach((element) => {
+  listEmployees.appendChild(element);
```

Comment:
This code is directly manipulating the DOM inside a loop, which can be inefficient and lead to performance issues. It would be better to create a document fragment, append the elements to the fragment, and then append the fragment to the DOM once all the elements have been added.